src/WebUI/ImageDetails/ImageDetails.tsx

```diff
@@ -61,6 +62,9 @@ export class ImageDetails extends BaseComponent<IImageDetailsProperties, IImageD
                             {this._displayImageName}
                         </HeaderTitle>
                     </HeaderTitleRow>
+                    <HeaderDescription className={"text-ellipsis"}>
+                        {jobName}
+                    </HeaderDescription>
```

Comment:
singleClickActivation is false, still on hover 'hand' is shown for cursor. we should check why this is happening.